File: 1° Período - Introdução a Programação 1/Exercicios em Python/Lista 4 - Python/q7.py
#Faça um algoritmo que leia do usuário cinco números. Se os valores forem diferentes identifique e
#imprima o maior e o menor valor.

x=int(input("Insira o número: "))
maior=x
menor=x
for i in range(4):
    x=int(input("Insira o número: "))
    if x>maior:
        maior=x
    else:
        if x<menor:
            menor=x
print("Maior: ", maior, "\nMenor: ", menor)
        
        

File: 1° Período - Introdução a Programação 1/Exercicios em C/Lista 4 - C/q7.c
//Faça um algoritmo que leia do usuário cinco números. Se os valores forem diferentes identifique e
//imprima o maior e o menor valor.

#include<stdio.h>
int main(){
	int i, x, maior, menor;
	printf("Insira um numero: ");
	scanf("%d", &x);
	maior=x;
	menor=x;
	for(i=0; i<3; i++){
		printf("Insira um numero: ");
		scanf("%d", &x);
		if(x>maior){
			maior=x;
		}
		if(x<menor){
			menor=x;
		}
	}
	printf("Maior: %d\nMenor: %d", maior, menor);
	system("pause");
}
